src/ja/worker/docker_events.py

```diff
@@ -0,0 +1,21 @@
+"""
+This module is responsible for the monitoring of the docker containers
+running on the worker machine.
+"""
+
+
+class DockerEvents:
+    """
+    Monitors the containers on the current machine
+    """
+
+    def __init__(self) -> None:
+        """!
+        Starts the event loop
+        that will be monitoring the state of the containers.
+        """
+
+    def change_filters(self) -> None:
+        """!
+        Change on witch events the server is going to be notified
```

Comment:
witch -> which

Reply:
Also, how is the change done? Should there be a parameter describing what events to listen for?